start.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+chown -rf root:root ~/.ssh/config
+chmod -rf 644 ~/.ssh/config
```

Comment:
can you help me here? i wonder how it is done in other projects, basically i am running into the permissions problems while mounting the ~/.ssh with proper keys